test/bin/start_webserver.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# This script should be run on the hypervisor to set up a caddy
+# file-server for the images used by Vms running test scenarios.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+echo "Starting web server in ${IMAGEDIR}"
+cd "${IMAGEDIR}"
+
+pkill caddy || true
+nohup caddy file-server --browse --listen "0.0.0.0:8080" --root "${IMAGEDIR}" --access-log &
```

Comment:
Why do we need access logs? It slows down the server.

Reply:
Since we're running this on the main hypervisor host, I think we should use another port. 
The chances of 8080 being taken are very high.

Reply:
I added a variable to adjust this if we need to.